hazelcast-integration/kubernetes/samples/ssl/README.md

```diff
@@ -0,0 +1,206 @@
+# Hazelcast with SSL (Enterprise Only)
+
+This a complete example presenting how to use Hazelcast Enterprise with SSL enabled on the connections between Hazelcast Members and Hazelcast Client.
+
+## Introduction
+
+This example focuses on the security features and assumes that you have some general knowledge about:
+ * Hazelcast on Kubernetes:
+   * [Hazelcast Kubernetes README](https://github.com/hazelcast/hazelcast-kubernetes)
+   * [Hazelcast Kubernetes Code Sample](../../)
+   * [Hazelcast Kubernetes Embedded Code Sample](../embedded)
+ * Java KeyStore and TrustStore
+   * [Oracle's introduction to KeyStore and TrustStore](https://docs.oracle.com/cd/E19509-01/820-3503/6nf1il6er/index.html)
+   * [How KeyStore and TrustStore were Generated](#how-keystore-and-truststore-were-generated)
+
+The example also assumes you have a running Kubernetes cluster and the `kubectl` tool installed.
+
+## 1. Creating Hazelcast Cluster
+
+Hazelcast must have access to `keystore` and `truststore`, so you have to either mount a volume with these files or include them in the Docker image itself. The second solution is simpler to describe, so let's use it here. The prepared `Dockerfile` already includes the commands to add `keystore` and `truststore` as well as `hazelcast.xml` with SSL enabled. Therefore, all you need to do is to build and push it into your Docker Hub.
```

Comment:
I felt like seeing mount option here and also in the Note marker as duplicate. I think we can remove mount mention here.

Reply:
Good idea, I'll remove it.